docs/plugins/components.md

```diff
@@ -0,0 +1,842 @@
+---
+sidebar_position: 5
+title: Components
+description: Documented components for plugins
+---
+
+# Components
+
+:::caution
+
+Any components below whose name ends in `Item` already has a `FormItem` (or equivalent) wrapper. You
+should not use this on those components.
+
+:::
+
+## Input components
+
+All input components ending with `Item` have their original counterpart, and differ in some props:
+
+| Name        | Type                | Default | Description                                      |
+| ----------- | ------------------- | ------- | ------------------------------------------------ |
+| `children?` | React.ReactNode     |         | Wrapper component title                          |
+| `note?`     | string              |         | Description of what the input component is about |
+| `style?`    | React.CSSProperties |         | Wrapper component style                          |
+
+### Checkbox and CheckboxItem
+
+Example:
+
+```tsx
+import { components } from "replugged";
+const { CheckboxItem } = components;
+
+export function Settings(): React.ReactElement {
+  return <CheckboxItem {...util.useSetting(cfg, "foo", "bar")}>Checkbox title</CheckboxItem>;
+}
+```
+
+Properties:
+
+| Property  | Description                                                     |
+| --------- | --------------------------------------------------------------- |
+| `.Aligns` | Checkbox alignments <br/>`TOP` \| `CENTER`                      |
+| `.Shapes` | Checkbox shapes <br/>`BOX` \| `ROUND` \| `SMALL`                |
+| `.Types`  | Checkbox types <br/>`DEFAULT` \| `INVERTED` \| `GHOST` \| `ROW` |
```

Comment:
Are these available on Checkbox or CheckboxItem or both?

Reply:
I had thought about it too, what if we set all enums on the wrapped components we have, so they are accessible from both comps?

Reply:
I think that's a good idea. Either way, this page should clarify what the properties are on somehow (could be a note at the top).